packages/core/src/version.ts

```diff
@@ -1,2 +1,2 @@
 // generated by genversion
-export const version = '1.8.0';
+export const version = '1.8.3';
```

Comment:
This was not committed in git since migrating to yarn v3, we have a task to fix this.
However this was still updated when we actually made the releases, so there is no impact on versions `1.8.1` and `1.8.2`.